package.json

```diff
@@ -6,13 +6,15 @@
   "scripts": {
     "compile": "tsc",
     "copy-graphql": "mkdir ./dist/schemas && cp ./src/schemas/*.graphql ./dist/schemas/",
-    "build": "rm -rf ./dist && npm run compile",
+
     "generate": "graphql-codegen",
-    "dev": "ts-node-dev --respawn ./src/index.ts",
+    "lint": "npx eslint . --ignore-pattern dist/ --ignore-pattern src/types.ts",
     "test": "jest --coverage",
-    "test-tsc": "jest -c ./jest.tsc.config.js",
-    "lint": "npx eslint .",
+
+    "build": "rm -rf ./dist && npm run lint && npm run generate && npm run test && npm run compile",
```

Comment:
I wonder if "npm run lint" and "npm run test" should be run separately in our pipeline. That way we can run the linting and testing first, and only run the build if the linting or tests pass.